packages/lib/src/artifacts/ZWeb3.ts

```diff
@@ -10,6 +11,34 @@ const NETWORKS = {
   42: 'kovan'
 };
 
+export interface Web3Transaction {
+  hash: string;
+  nonce: number;
+  blockHash: string;
+  blockNumber: number;
+  transactionIndex: number;
+  from: string;
+  to: string;
+  value: BN;
+  gasPrice: BN;
+  gas: number;
+  input: string;
+}
+
+export interface Web3TransactionReceipt {
```

Comment:
I'm concerned about these interfaces, as they depend on external libraries that could change their structure in the future (just assuming things here, but I'm pretty sure `web3` has changed a lot from 0.x to 1.y). I would suggest to only define interfaces related to our classes, functions, etc. (note that `Web3TransactionReceipt` and `Web3Transaction` have a strong dependence with `web3`).

Reply:
Agreed, removing the interfaces.